distribution/src/config/jvm.options

```diff
@@ -89,3 +89,6 @@ ${error.file}
 # See please https://bugs.openjdk.org/browse/JDK-8341127 (openjdk/jdk#21283)
 23:-XX:CompileCommand=dontinline,java/lang/invoke/MethodHandle.setAsTypeCache
 23:-XX:CompileCommand=dontinline,java/lang/invoke/MethodHandle.asTypeUncached
+
+# It should be JDK-24 (but we cannot bring JDK-24 since Gradle does not support it yet)
+21-:-javaagent:agent/opensearch-agent-3.0.0-SNAPSHOT.jar
```

Comment:
Should we not hardcoding SNAPSHOT here?

Reply:
> Should we not hardcoding SNAPSHOT here?

@peterzhuamazon absolutely not, this is a POC (not supposed to be merged as is) with a few shortcuts like that, thank you